packages/strategies/stress.go

```diff
@@ -179,62 +155,75 @@ func (s *StressTest) ValidateBlock(ctx context.Context, e *eth.ExecutionPayloadE
 	return nil
 }
 
+// Adds new eth transfer txs to the block until the target gas usage is met
 func (s *StressTest) packItUp(input *types.Block) types.Transactions {
 	gasInfo := input.Transactions()[len(input.Transactions())-1]
 
-	originTransactionUse := input.GasUsed()
-	targetUsage := originTransactionUse * 2
+	originalGasUsed := input.GasUsed()
+	targetUsage := s.cfg.GasTarget
 
-	fillUp := targetUsage - originTransactionUse
+	fillUpRemaining := int64(targetUsage)
```

Comment:
This is a key bug fix. `fillUp`/`fillUpRemaining` must be an `int64` instead of `uint64` otherwise the `for` loop below can get stuck due to this logic:
```
		if fillUpRemaining < gasUsed {
			break
		}
```